docs/excel/excel-add-ins-workbooks.md

```diff
@@ -338,6 +338,40 @@ The Excel API also lets add-ins turn off calculations until `RequestContext.sync
 context.application.suspendApiCalculationUntilNextSync();
 ```
 
+## Detect workbook activation (preview)
+
+> [!NOTE]
+> The `Workbook.onActivated` event is currently only available in public preview. [!INCLUDE [Information about using preview APIs](../includes/using-excel-preview-apis.md)]
+> 
+
+Your add-in can detect when a workbook is activated. After a workbook is opened, it will become "inactive" if the user switches focus to another workbook, to another application, or (in Excel on the web) to another tab of the web browser. A workbook is "activated" when the user switches focus back to this workbook. After detecting workbook activation, your add-in can trigger callback functions, such as refreshing workbook data.
```

Comment:
Some phrasing suggestions. Also, I think that bold or italics are better for emphasis, rather than quotes.
```suggestion
Your add-in can detect when a workbook is activated. A workbook becomes **inactive** when the user switches focus to another workbook, to another application, or (in Excel on the web) to another tab of the web browser. A workbook is **activated** when the user returns focus to the workbook. The workbook activation can trigger callback functions your add-in, such as refreshing workbook data.
```

Reply:
The quotation marks are intended to indicate that these words are part of the API name. The [Style Guide](https://styleguides.azurewebsites.net/Styleguide/Read?id=2700&topicid=28768) is kind of ambiguous about whether quotation marks are recommended in this type of scenario. I don't want to use bold for emphasis because that usually represents content in the UI (a UI button or worksheet name). I will try switching to italics.